polling_stations/apps/file_uploads/api.py

```diff
@@ -58,8 +59,11 @@ def save(validated_data):
             if (
                 file_set.first().ems == "Democracy Counts" and file_set.count() == 2
             ) or file_set.first().ems != "Democracy Counts":
-                upload.make_pull_request()
+                upload.upload_complete(user=self.context["request"].user)
 
+            else:
+                # TO DO How to handle this?
```

Comment:
I think by 'this' you mean the case where there is no `file_set`. 
The case where there is no `file_set` should throw an error earlier as shown in the tests. If there is a way of getting here without throwing an error then I guess the best handling would be to send us an email telling us that something has gone wrong.

However, at the moment your `else` is corresponding to the `if` statement which compares the ems of the `file_set`:
```
            if (
                file_set.first().ems == "Democracy Counts" and file_set.count() == 2
            ) or file_set.first().ems != "Democracy Counts":
```
Not `if file_set.exists()`. Again I thin this case is handled, but there could be an email to let us know.

Reply:
Thanks for catching this. I've added an email trigger an `else` statement focused on the presence of `file_set` in f1b3abced01d79813793a0a78f97ff5c3a760dea